media-tv/w_scan/w_scan-20170107.ebuild

```diff
@@ -13,12 +13,13 @@ KEYWORDS="~amd64 ~x86"
 IUSE="doc examples"
 
 DEPEND=">=virtual/linuxtv-dvb-headers-5.8"
-RDEPEND=""
 
-src_install() {
-	emake DESTDIR="${D}" install
+PATCHES=( "${FILESDIR}"/${P}-fno-common.patch )
+DOCS=( ChangeLog README )
 
-	dodoc ChangeLog README
+src_install() {
+	default
+	emake DESTDIR="${ED}" install
```

Comment:
you sure this one is needed (check the definition of `default_src_install`)?

Reply:
Nope, removed it, thanks